### ISSUE TITLE
train-loss = Nan

### ISSUE THREAD

**Issue by xiyufeng2** [jbwang1997/OBBDetection#100]
2021-12-20 17:13:30,133 - mmdet - INFO - Epoch [12][150/4867]   lr: 5.000e-05, eta: 0:29:00, time: 0.400, data_time: 0.005, memory: 4601, loss_rpn_cls: nan, loss_rpn_bbox: nan, loss_cls: nan, acc: 11.5102, loss_bbox: nan, loss: nan, grad_norm: nan
all loss is always nan.what's wrong?

**Comment by jbwang1997** [comment]
Hello, can you provide more information?

**Comment by xiyufeng2** [comment]
I didn't load pretrained weight to train the model , leading the loss to be large. I think the weight init method is not suitable
